src/main/java/com/exadel/discount/service/impl/FavoriteServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package com.exadel.discount.service.impl;
+
+import com.exadel.discount.dto.favorite.CreateFavoriteDto;
+import com.exadel.discount.dto.favorite.FavoriteDto;
+import com.exadel.discount.entity.Discount;
```

Comment:
You don't have this

Reply:
Just comment all parts with missing classes temporarily

Reply:
But I will soon. Otherwise it will not work anyway